crates/sui-types/src/gas.rs

```diff
@@ -340,29 +357,32 @@ impl<'a> SuiGasStatus<'a> {
         self.deduct_computation_cost(&cost)
     }
 
+    pub fn charge_computation_gas_for_storage_mutation(
+        &mut self,
+        size: u64,
+    ) -> Result<(), ExecutionError> {
+        // Computation cost of a mutation is charged based on the sum of the old and new size.
```

Comment:
I think it doesn't make a lot of sense to keep this comment unless we're passing in `old_size` and `new_size` as separate args

Reply:
removed